.husky/pre-commit

```diff
@@ -1 +1,4 @@
-yarn precommit
```

Comment:
Why this change? Can you please reference some docs on why the husky command needs to be added?

Also, we don't use `npm` in the repo, only `yarn`. And in order to manage what needs to be run it's better to maintain that in the `package.lock` file rather then here in the husky file. That's why we run `yarn precommit`, which could run `yarn test` or anything else.

Reply:
My local husky configuration had some issues finding the `yarn` command, so I used `npm` instead. Ultimately, this change was pushed by mistake.